Header header
bool wheeldropCaster
bool wheeldropLeft
bool wheeldropRight
bool bumpLeft
bool bumpRight
bool wall
bool cliffLeft
bool cliffFronLeft
bool cliffFrontRight
bool cliffRight
bool virtualWall
uint8 infraredByte
bool advance
bool play
int16 distance
int16 angle
uint8 chargingState
uint16 voltage
int16 current
int8 batteryTemperature
uint16 batteryCharge
uint16 batteryCapacity
uint16 wallSignal
uint16 cliffLeftSignal
uint16 cliffFrontLeftSignal
uint16 cliffFrontRightSignal
uint16 cliffRightSignal
bool homeBase
bool internalCharger
uint8 songNumber
uint8 songPlaying
